ssl/ssl_locl.h

```diff
@@ -1422,6 +1422,8 @@ struct ssl_st {
     size_t num_tickets;
     /* The number of TLS1.3 tickets actually sent so far */
     size_t sent_tickets;
+    /* The next nonce value to use when we send a ticket on this connection */
+    size_t next_ticket_nonce;
```

Comment:
Is `size_t` the right type to use here, given that it has a different max value 32-bit vs 64-bit?

Reply:
This (and other definitions of `nonce`) should be `uint64_t`, since:
```
#define TICKET_NONCE_SIZE       8
```
Otherwise, this will be a problem on 32-bit.

Reply:
Good point. Probably it should be uint64_t